packages/design-toolkit/src/components/flashcard/index.tsx

```diff
@@ -0,0 +1,171 @@
+import { DetailsListLabel } from '../details-list/label';
+import { DetailsListValue } from '../details-list/value';
+import { Skeleton } from '../skeleton';
+import { FlashcardStyles } from './styles';
+import type {
+  FlashcardComponentProps,
+  FlashcardDetailsListProps,
+  FlashcardProps,
+} from './types';
+
+const {
+  container,
+  hero,
+  header,
+  subHeader,
+  flashcardData,
+  detailsList,
+  detailsLabel,
+  detailsValue,
+  skeleton,
+} = FlashcardStyles();
+
+export const FlashcardContext = createContext<FlashcardProps>({
+  isLoading: false,
+});
+
+/**
+ * Example usage.
+ *
+ * <Flashcard isLoading={isLoading}>
+ *  <FlashcardHero>
+ *    <FlashcardHeader>
+ *      {header}
+ *    </FlashcardHeader>
+ *    <FlashcardSubheader>
+ *      {subHeader}
+ *    </FlashcardSubheader>
+ *  </FlashcardHero>
+ *  <FlashcardAdditionalData>
+ *    {secondaryData}
+ *  </FlashcardAdditionalData>
+ *  <FlashcardDetailsContainer details={details} />
+ * </Flashcard>
+ */
+export function Flashcard(props: FlashcardProps) {
+  const { isLoading, children, className, ...rest } = props;
+
+  return (
+    <FlashcardContext.Provider value={{ isLoading }}>
+      <div {...rest} className={container({ className })}>
+        {children}
+      </div>
+    </FlashcardContext.Provider>
+  );
+}
+Flashcard.displayName = 'Flashcard';
+
+export function FlashcardHero(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+  const { isLoading } = useContext(FlashcardContext);
+
+  if (isLoading) {
+    return (
+      <div {...rest} className={hero({ className: 'gap-s' })}>
+        <Skeleton className={skeleton()} />
+        <Skeleton className={skeleton({ className: 'max-w-1/2' })} />
+      </div>
+    );
+  }
+
+  return (
+    <div {...rest} className={hero()}>
+      {children}
+    </div>
+  );
+}
+FlashcardHero.displayName = 'FlashcardHero';
+
+export function FlashcardHeader(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+
+  return (
+    <div {...rest} className={header({ className })}>
+      {children}
+    </div>
+  );
+}
+FlashcardHeader.displayName = 'FlashcardHeader';
+
+export function FlashcardSubheader(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+
+  return (
+    <div {...rest} className={subHeader({ className })}>
+      {children}
+    </div>
+  );
+}
+FlashcardSubheader.displayName = 'FlashcardSubheader';
+
+export function FlashcardAdditionalData(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+  const { isLoading } = useContext(FlashcardContext);
+
+  // While loading, don't display.
+  if (isLoading) {
+    return null;
+  }
+
+  return (
+    <div {...rest} className={flashcardData({ className })}>
+      {children}
+    </div>
+  );
+}
+FlashcardAdditionalData.displayName = 'FlashcardAdditionalData';
+
+export const FlashcardDetailsList = (props: FlashcardDetailsListProps) => {
```

Comment:
missing displayName for this one. Also, every other component uses `function` sytax versus `const`/arrow. Worth consistency of one pattern, so I'd suggest updating this to be `function`